src/bin/miri.rs

```diff
@@ -282,6 +282,16 @@ fn main() {
                     };
                     miri_config.tracked_alloc_id = Some(miri::AllocId(id));
                 }
+                arg if arg.starts_with("-Zmiri-max-yield-iterations=") => {
```

Comment:
New options should come with a documentation in the README. That's also a good opportunity to try and give a short but precise description of what this does. ;)